src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,164 @@
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### Combining Filters
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### iOS Frosted Notifications
+
+```css
+.frost {
+  backdrop-filter: blur(.25rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ios_notifications.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://s.codepen.io/robinrendle/pen/ZoXGgG" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Conclusion
+Better late than never! Safari's had this feature since 2015. Regardless, Chrome's excited to release it in 76. The [Chromium bug](https://crbug.com/497522) has **560+ stars** on it, clearly marking it as a desired web design capability.
+
+Here's a final CSS API list and additional resources in case you're ready to start developing.
+
+### Overview
+```css
+/* Keyword value */
+backdrop-filter: none;
+
+/* URL to SVG filter */
+backdrop-filter: url(commonfilters.svg#filter);
+
+/* <filter-function> values */
+backdrop-filter: blur(2px);
+backdrop-filter: brightness(60%);
+backdrop-filter: contrast(40%);
+backdrop-filter: drop-shadow(4px 4px 10px blue);
+backdrop-filter: grayscale(30%);
+backdrop-filter: hue-rotate(120deg);
+backdrop-filter: invert(70%);
+backdrop-filter: opacity(20%);
+backdrop-filter: sepia(90%);
+backdrop-filter: saturate(80%);
+
+/* Multiple filters */
+backdrop-filter: url(filters.svg#filter) blur(4px) saturate(150%);
+
+/* Global values */
+backdrop-filter: inherit;
+backdrop-filter: initial;
+backdrop-filter: unset;
+```
+
+### One more demo
+A glimpse into the now possible in Chrome:
+
+<figure class="w-figure w-figure--fullbleed">
+  {% YouTube 'MWHBpReeAJ0' %}
+  <figcaption class="w-figcaption">
+    Demo by <a href="https://ferdychristant.com/please-help-make-backdrop-filter-a-reality-f81805ba3d52" target="_blank">Ferdy Christant</a>
+  </figcaption>
+</figure>
+
+### Additional Resources
+- [Specification](https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty)
+- [Proposal](https://github.com/w3ctag/design-reviews/issues/353)
+- [GitHub](https://github.com/mfreed7/backdrop-filter-feature)
```

Comment:
What is this a github link to? Can you include that in the link text?

Reply:
It's documentation for the feature. I would remove it. When we're the first implementor, such links are usually the only content that exists. Since we're late to the game, we need to devote our attention to making the MDN page accurate and up to date.

Reply:
cool, github link is gone!